ui/app/templates/components/dashboard/secrets-engines-card.hbs

```diff
@@ -27,16 +27,12 @@
               <div>
                 <div class="is-flex-center">
                   {{#if backend.icon}}
-                    <ToolTip @horizontalPosition="left" as |T|>
-                      <T.Trigger>
-                        <Icon @name={{backend.icon}} class={{unless backend.isSupportedBackend "has-text-grey"}} />
-                      </T.Trigger>
-                      <T.Content @defaultClass="tool-tip">
-                        <div class="box">
-                          {{or backend.engineType backend.path}}
-                        </div>
-                      </T.Content>
-                    </ToolTip>
+                    <Icon
+                      @name={{backend.icon}}
+                      @title="{{or backend.engineType backend.path}} type backend"
+                      title={{or backend.engineType backend.path}}
```

Comment:
```suggestion
                      @title="{{or backend.engineType backend.path}} type backend"
```
The extra title might have been copy pasta?

Reply:
Nope, this is intentional! Since I removed the Tooltip (which was an a11y violation) I added this title tag so that it shows up on hover for sighted users

Reply:
Makes sense! thanks for clarifying that!